src/codeflare_sdk/cli/commands/define.py

```diff
@@ -15,25 +17,57 @@ def cli():
 @click.pass_context
 @click.option("--name", type=str, required=True)
 @click.option("--namespace", "-n", type=str)
-@click.option("--head_info", cls=PythonLiteralOption, type=list)
-@click.option("--machine_types", cls=PythonLiteralOption, type=list)
-@click.option("--min_cpus", type=int)
-@click.option("--max_cpus", type=int)
-@click.option("--min_worker", type=int)
-@click.option("--max_worker", type=int)
-@click.option("--min_memory", type=int)
-@click.option("--max_memory", type=int)
+@click.option("--head-info", cls=PythonLiteralOption, type=list)
+@click.option("--machine-types", cls=PythonLiteralOption, type=list)
+@click.option("--min-cpus", type=int)
```

Comment:
I think this will have to change to `--num-cpus` once this is rebased on the current upstream.